packages/get_line/get_line.2.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "ligand@free.fr"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/get_line"
+dev-repo: "git://github.com/UnixJunkie/get_line"
+bug-reports: "https://github.com/UnixJunkie/get_line/issues"
+license: "GPL"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build"]
+]
+install: [
+  ["cp" "dist/build/get_line/get_line" bin]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/get_line"]
+]
+depends: [
+  "obuild" {build}
+  "core"
+  "batteries"
```

Comment:
This [appears to be missing a dependency on `Core_kernel`](https://travis-ci.org/ocaml/opam-repository/jobs/207486768):

```
# File "get_line.ml", line 4, characters 2-35:
# Error: Unbound module Core_kernel
```